tests/README.md

```diff
@@ -34,10 +34,37 @@ Usage: test.sh [--help | --list] [ --from-test <test-function-name> ] [test_all]
     This will resume test execution from the `test-build-and-install-sev-snp-simulator`
     test to run the remaining tests in the order reported by the `--list` argument.
 
+## Executing test targets: `--list` output
```

Comment:
Elaborated usage of test-targets in `test.sh`. 

See this [tests/README](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/agurajada/214-py-private-key/tests/README.md) or better rendition.